### ISSUE TITLE
Please update for flutter 3.+

### ISSUE THREAD

**Issue by ProFive** [Sub6Resources/chewie_audio#19]
playedColor: Theme.of(context).colorScheme.secondary,
                  handleColor: Theme.of(context).colorScheme.secondary,
                  bufferedColor: Theme.of(context).colorScheme.background,

**Comment by jeff9315** [comment]
I got it to work by changing lines 241 - 246 of 
C:\Users\ ......... \AppData\Local\Pub\Cache\hosted\pub.dev\chewie_audio-1.3.0\lib\src\material_controls.dart
to

````
ChewieProgressColors(
                  playedColor: Theme.of(context).colorScheme.secondary,
                  handleColor: Theme.of(context).colorScheme.secondary,
                  bufferedColor: Theme.of(context).backgroundColor,
                  backgroundColor: Theme.of(context).disabledColor),
        ),
```

**Comment by Sub6Resources** [comment]
Fixed in #18
